components/search/index.css

```diff
@@ -41,43 +169,168 @@ governing permissions and limitations under the License.
+
+    &::placeholder {
+      color: var(--highcontrast-search-color-disabled, var(--mod-search-color-disabled, var(--spectrum-search-color-disabled)));
+    }
+  }
 }
 
-.spectrum-Search-textfield {
-  &:after {
-    border-radius: var(--spectrum-alias-search-border-radius);
+/* Focus Indicator (Ring) */
+.spectrum-Search:not(.spectrum-Search--quiet) .spectrum-Search-textfield {
+  &::after {
+    border-radius: calc(
+      var(--mod-search-border-radius, var(--spectrum-search-border-radius)) + 
+      var(--mod-search-border-width, var(--spectrum-search-border-width)) +
+      var(--mod-search-focus-indicator-gap, var(--spectrum-search-focus-indicator-gap))
+    );
+    block-size: calc(100% + (var(--mod-search-focus-indicator-gap, var(--spectrum-search-focus-indicator-gap)) * 2));
+    inline-size: calc(100% + (var(--mod-search-focus-indicator-gap, var(--spectrum-search-focus-indicator-gap)) * 2));
+    margin-block-start: calc(
+      (var(--mod-search-focus-indicator-gap, var(--spectrum-search-focus-indicator-gap)) + 
+      var(--mod-search-focus-indicator-thickness, var(--spectrum-search-focus-indicator-thickness)))
+      * -1
+    );
+    margin-inline-start: calc(
+      (var(--mod-search-focus-indicator-gap, var(--spectrum-search-focus-indicator-gap)) + 
+      var(--mod-search-focus-indicator-thickness, var(--spectrum-search-focus-indicator-thickness)))
+      * -1
+    );
+  }
+
+  &.is-keyboardFocused,
+  &:focus-within {
+    &::after {
+      border-width: var(--mod-search-focus-indicator-thickness, var(--spectrum-search-focus-indicator-thickness));
+      border-color: var(--highcontrast-search-focus-indicator-color, var(--mod-search-focus-indicator-color, var(--spectrum-search-focus-indicator-color)));
+    }
   }
 }
 
+/* Standard / Default */
 .spectrum-Search:not(.spectrum-Search--quiet) {
   .spectrum-Search-icon {
-    /* todo: when we add t-shirt sizing, this will be wrong ❤️ */
-    inset-inline-start: var(--spectrum-alias-search-padding-left-m);
+    inset-inline-start: var(--mod-search-edge-to-visual, var(--spectrum-search-edge-to-visual));
   }
+
   .spectrum-Search-input {
-    /* todo: when we add t-shirt sizing, this will be wrong ❤️ */
     padding-inline-start: calc(
-      var(--spectrum-alias-search-padding-left-m) +
-        var(--spectrum-alias-workflow-icon-size-m) +
-        var(--spectrum-textfield-m-texticon-icon-gap) -
-        var(--spectrum-textfield-m-texticon-border-size)
+      var(--mod-search-edge-to-visual, var(--spectrum-search-edge-to-visual)) +
+      var(--mod-search-icon-size, var(--spectrum-search-icon-size)) +
+      var(--mod-search-text-to-icon, var(--spectrum-search-text-to-icon))
     );
+    padding-inline-end: var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size));
+  }
+
+  &:not(:has(.spectrum-Search-clearButton)) .spectrum-Search-input {
```

Comment:
I wasn't sure about this addition. The design shows spacing on the right of the input without the clear button using this token. I thought this might be a nice enhancement in the case of someone not wanting to use the clear button. Browsers without `:has` support would just have a little extra right padding. If that use case is not necessary to support, I could remove this CSS. If it is necessary to support and we want greater than `:has` support, then we'd probably need to add another parent class (e.g. has-reset).

Reply:
I think this bit:

> case of someone not wanting to use the clear button

is a good question for design folks. "Can people use this without the clear button?"

I don't mind the thought of this, though! And yes, you're right: we're not yet able to fully support/use `:has` for critical UI, but if folks who use browsers that don't yet support `:has` don't really see a difference, I think this is a fine progressive enhancement.

Reply:
I've removed this, per our discussion, and the discussion with the designer. The component shouldn't be used without the clear button, so that use case does not need to be supported right now.